versions.yaml

```diff
@@ -263,6 +263,32 @@ externals:
       .*/v?(\d\S+)\.tar\.gz
     version: "v1.0.0-rc5"
 
+  cmake:
+    description: |
+      Build system, to build grpc-rs.
+    url: "https://github.com/Kitware/CMake"
+    uscan-url: >-
+      https://github.com/Kitware/CMake/releases/download/
+      v?([\d\.]+)/cmake-([\d\.]+)\.tar\.gz
+    version: "3.15.3"
+    meta:
+      description: |
+        'newest-version' is the latest version works.
+      newest-version: "3.15.3"
+
+  musl:
+    description: |
+      musl is used to build rust agent.
+    url: "https://www.musl-libc.org/"
+    uscan-url: >-
+      https://www.musl-libc.org/releases/
+      musl-([\d\.]+)\.tar\.gz
+    version: "1.1.23"
+    meta:
+      description: |
+        'newest-version' is the latest version works.
```

Comment:
Nit: `s/version works/version known to work/` maybe.

Reply:
updated -_-